### DISCUSSION TITLE
UniFi Network Server LXC script issue

### DISCUSSION THREAD

**Discussion by martin8310** [tteck/Proxmox#3850] [Help Needed]
Does anyone know why my installation aborts with the error message:

`[ERROR] in line 32: exit code 0: while executing command gpg --dearmor > /usr/share/keyrings/mongodb-server-7.0.gpg`

What settings are you currently utilizing: Advanced

regards
Martin

**Comment by tteck** [comment]
What CPU are you using?
> 🛈 CPU with AVX Instruction Set required

**Reply by martin8310** [reply]
Thanks, I discovered my mistake. The IP address was assigned twice. Sorry
